src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -373,7 +373,8 @@ public Borrower getBorrowerFromId(BorrowerId borrowerId) {
 
     @Override
     public void setBorrower(Borrower borrowerToEdit, Borrower editedBorrower) {
-
+        borrowerRecords.removeBorrower(borrowerToEdit);
+        borrowerRecords.addBorrower(editedBorrower);
```

Comment:
We should also edit the borrower in the borrowersMap

Reply:
okay!

Reply:
> We should also edit the borrower in the borrowersMap
the addBorrower method adds borrower in both listOfBorrowers and borrowersMap already. Is it what you mean?

Reply:
You're right! My bad haha